pkg/apis/triggers/v1alpha1/register.go

```diff
@@ -32,17 +32,8 @@ const (
 	// EventIDLabelKey is used as the label identifier for an EventListener event.
 	EventIDLabelKey = "/triggers-eventid"
 
-	// TriggerBindingLabelKey is used as the label identifier for a TriggerBinding.
-	TriggerBindingLabelKey = "/triggerbinding"
-
-	// TriggerTemplateKey is used as the label identifier for a TriggerTemplate
-	TriggerTemplateKey = "/triggertemplate"
-
 	// TriggerLabelKey is used as the label identifier for a Trigger
 	TriggerLabelKey = "/trigger"
-
-	// LabelEscape is an escaped GroupName safe for use in resource types.
-	LabelEscape = "tekton\\.dev"
```

Comment:
I'm a bit confused by these being deleted: 1) were we not using them before? 2) should we be?

Reply:
We needed it before when we were using sjson to set the labels...now we don't the escaping anymore 😄